### ISSUE TITLE
UX/UI enhancement

### ISSUE THREAD

**Issue by project-13** [upscayl/upscayl#414]
### Checklist

- [X] I have checked that this issue isn't addressed in [the Wiki](https://github.com/upscayl/upscayl/wiki).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.

### Description

Hi team,
Thank you for all the hard work you put in daily.
From using upscayl, i can suggest some small changes to enhance the ux/ui of the app :
The goal is to win some space (no scroll bar or less scrolling), to squish similar properties and perform actions in less clicks.

### Related Problem(s)

Details :

1. Delete unnecessary text (step1, step2, step3, select upscaling type...)
2. output folder : 
delete the "save output folder (permanently)" check box
replace the text "default to image's path" by a checkbox "save to image's path", if unchecked show the button "set the output folder" 
The output folder must be saved by default and the status of the checkbox must be saved by default
loading a new image mustn't change the value of the default output folder or the checkbox status
3. Image scale :
delete the double upscale checkbox
move the image scale cursor from setting tab to upscayl tab
add more values to the cursor to integrate the double upscale
before (2X 3X 4X)
after (2X 3X 4X 6X 9X 8X 12X 16X)
(PS : We can get values that we can't get by the checkbox double upscaling 6X, 8X, 12X : 6X = upscaling 2X then 3X or 3X then 2X)
4. Optional : the batch upscale can have its own tab instead of a checkbox

please find below an illustration of these changes
![upscaly](https://github.com/upscayl/upscayl/assets/35433969/829e79a6-1272-4d12-9864-503e06014fc2)

Thank you

**Comment by aaronliu0130** [comment]
1. The "Double Upscayl" toggle is different from the scale factor.
2. Having a lot of different types of buttons right together (dropdowns, file selection) without headings can be overwhelming and maybe confusing.
3. The "Save Output Folder Permanently" checkbox means that it'll remember the output folder you set across restarts of the Upscayl application, and it's a feature. It is a bit unclear though, maybe the label should be changed to "Remember Output Folder".
4. What if the "Save to image's path" checkbox is unchecked but the user doesn't select a new output folder either?
5. The "Double Upscayl" toggle is different from the scale factor. It will Upscayl the Upscayl'd image while the scale selection tries to make the model directly output that scale. They are separate features and cannot be easily integrated, not even with a 16x scale factor.
6. The scale selection currently doesn't work on our bundled models. Putting it in the main screen can cause users to think it basically works, and we already have duplicate reports that it doesn't work. It should be kinda hidden as it doesn't work yet.
7. The Batch Upscayl uses essentially the same UI as the main Upscayl but with different text. It does not hold the same distinction between Upscayl and Settings.

**Comment by project-13** [comment]
Thanks for your reply.

1- From the user view, the result is the same. Moreover, instead of double, the appscayl magic can be ran twice but with different upscaling levels to get values we can't get with a double upscaling (6X, 8X, 12X) (cf. point 5)
2- Buttons are self-explained by their labels
3- the path is saved but it's never used, when the user select an image, the output folder changes automatically to the selected image's folder
4- it can be saved in the selected input's folder
5- from what i understand the app behaviour is like this:
- simple upscale : input => input * X (X : 2, 3, 4)
- double upscale checked : simple upscale then simple upscale the upscaled image with the same factor, at the end the factor is X^2 (X^2: 4, 9, 16) :
 input => input * X => (input * X) *X 
It will be more clear and more efficient to allow the user select the final upscale factor, the app can execute the upscale magic once or twice :
when the user selects 2X, 3X or 4X => the app runs the simple upscale
when the user selects 6X, 9X, 12X or 16X => the app runs simple upscale with a factor X (X : 2, 3 or 4) then it reruns a simple upscale with a factor Y different from X (Y: 2, 3 or 4) on the upscaled image
Basically, no major change
PS: the temporary generated image from the first upscale can be deleted
6- those are just some UX/UI suggestions, it can be implemented, modified, refused or programmed.
7- in terms of functionnality, it's a major one, it deserves to have it's own tab (even if the executed source code is the same as the other tab with différent labels or duplicated) instead of a toggle button

**Comment by aaronliu0130** [comment]
I wonder which parts of this were completed.

**Comment by NayamAmarshe** [comment]
> I wonder which parts of this were completed.

Just the option rename.
The rest, well, I considered them.
